apps/web-mzima-client/src/app/settings/surveys/survey-task/survey-task.component.html

```diff
@@ -83,16 +83,34 @@ <h2>
           [data-qa]="'field-' + field"
         >
           <div class="field-item__main">
-            <mat-icon
-              cdkDragHandle
-              class="field-item__non-drag-handle"
-              svgIcon="drag-handle"
-            ></mat-icon>
-            <span class="field-item__title" [data-qa]="'field-name'">{{
-              isDefaultLanguageSelected
-                ? field.label
-                : field.translations[selectLanguageCode]?.label || field.label
-            }}</span>
+            <div
+              class="field-item__main-inner"
+              (mouseenter)="trackHoverEvent($event, field)"
+              (mouseleave)="trackHoverEvent($event, field)"
+            >
```

Comment:
I think we need to think about the keyboard-users here as well, not just the hovering?

Reply:
This crossed my mind too. One reason why I added the videos about the 5 different options to the ticket, so that people can check it out, see and make comments @Angamanga 
The most accessible looking option is the one where all the icons show up already without having to hover. I've tried one or 2 things accessibility wise on this hover option we are going with, non is successful so far.

I will try again, and let you know how it goes:
> This works, with the mouse. Is there a way to make this available also for keyboard-users? 😬 I think perhaps with focus and blur-event + aria-labels? 🤔